plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/editors/groovy/ClassPathHandler.java

```diff
@@ -60,7 +61,8 @@ public static void addDirPathToPathList() {
 	    DirectoryDialog dialog = new DirectoryDialog(shell);
 	    String result = dialog.open();
 		try {
-			urls.add(new File(result).toURI().toURL());
+			URL url = new File("///d:" + result).toURI().toURL();
```

Comment:
Why do you have to use `"///d:"` as prefix ?

Reply:
URLClassLoader needs to get an url with this prefix to process this url as a directory.